app/lib/fake_data/complaint.rb

```diff
@@ -0,0 +1,59 @@
+require "ffaker"
+
+module FakeData
+  class Complaint
+    def data
+      @data ||= {
+        id: identifier,
+        type: "issues",
+        attributes: {
+          issueLastUpdated: datetime,
+          creationDate: datetime,
+          closedDate: datetime,
+          reopenedDate: datetime,
+          initialContactDate: datetime,
+          type: FFaker::Random.rand(2),
+          otherType: FFaker::Lorem.word,
+          issue: FFaker::Lorem.phrase,
+          priority: FFaker::Random.rand(3),
+          status: FFaker::Random.rand(3),
+          dueDate: FFaker::Time.date,
+          # these just sound cool for use as the grantee name :)
+          grantee: FFaker::AddressUK.street_name
+        },
+        relationships: {
+          grantAward: {
+            meta: {
+              id: identifier
+            }
+          },
+          grantProgram: {
+            meta: {
+              id: identifier
+            }
+          },
+          region: {
+            meta: {
+              id: identifier
+            }
+          }
+        },
+        links: {
+          self: "https://example.com/TODO",
+          html: "https://example.com/TODO"
+        }
+      }
+    end
+
+    def datetime
```

Comment:
i think these might be moved somewhere else at some point, as they are really just helpers, but i'd like to leave them close to the data until that's necessary